tests/benchmarks/sys_kernel/testcase.yaml

```diff
@@ -3,3 +3,7 @@ tests:
     arch_exclude: nios2 riscv32 xtensa
     min_ram: 32
     tags: benchmark
+  benchmark.kernel.qemu_cortex_m3:
```

Comment:
not sure how this is related to this PR.

Reply:
Some  of these patches are need to keep the sanity when we move to the new QEMU. 
So such patches need to be ready before we update the SDK with the new QEMU.